### DISCUSSION TITLE
Server support for Bitbucket

### DISCUSSION THREAD

**Discussion by Heretolearn-code** [modelcontextprotocol/.github#47] [Ideas]
### Pre-submission Checklist

- [X] I have verified this would not be more appropriate as a feature request in a specific repository
- [X] I have searched existing discussions to avoid duplicates

### Your Idea

can we make an imoplementation for bitbucket similar to Github. 

### Scope

- [X] Protocol Specification
- [ ] SDK Features
- [X] Documentation
- [ ] Developer Experience
- [ ] Other

**Comment by mar0der** [comment]
I think you could have the answer already if u have asked the same question to claude. Today i built python server for connecting and executing commands on linux server via ssh. Not single line written by me. It took 2-3 hours and some iterations

**Comment by jspahrsummers** [comment]
Closing this discussion as this would be more appropriate as a feature request.

Agree that you can get AI to write this for you as well! 😄
